share/functions/__fish_complete_mount_opts.fish

```diff
@@ -0,0 +1,174 @@
+function __fish_print_user_ids
+    if test -x /usr/bin/getent
+        for line in (getent passwd)
+            set v (string split : $line)
+            printf "%s\t%s\n" $v[3] $v[1]
+        end
+    end
+end
+
+function __fish_print_group_ids
+    if test -x /usr/bin/getent
+        for line in (getent group)
+            set v (string split : $line)
+            printf "%s\t%s\n" $v[3] $v[1]
+        end
+    end
+end
+
+function __fish_complete_mount_opts
+    set -l fish_mount_opts\
```

Comment:
Indentation is annoying on these following lines. Please run this through `fish_indent`.

Any weirdness that is left then is fish_indent's fault so that should be improved.